src/app_service/service/community_tokens/service.nim

```diff
@@ -258,6 +304,50 @@ QtObject:
       except Exception as e:
         error "Error processing Collectible deployment pending transaction event", msg=e.msg, receivedData
 
+    self.events.on(PendingTransactionTypeDto.DeployOwnerToken.event) do(e: Args):
+      var receivedData = TransactionMinedArgs(e)
+      try:
+        let deployState = if receivedData.success: DeployState.Deployed else: DeployState.Failed
+        let ownerTransactionDetails = toOwnerTokenDeploymentTransactionDetails(parseJson(receivedData.data))
+        if not receivedData.success:
+          error "Owner contracts not deployed", chainId=ownerTransactionDetails.ownerToken.chainId, address=ownerTransactionDetails.ownerToken.address
```

Comment:
Should we return or raise here?

Reply:
No, it means that transaction was failed, we need to follow the flow, and add token to database with Failed status.

Reply:
I will change this log into warn.